lib/cartopy/mpl/gridliner.py

```diff
@@ -95,8 +120,10 @@ class Gridliner(object):
     # maybe even a plain old mpl axes) and it will call the "_draw_gridliner"
     # method on draw. This will enable automatic gridline resolution
     # determination on zoom/pan.
-    def __init__(self, axes, crs, draw_labels=False, xlocator=None,
-                 ylocator=None, collection_kwargs=None):
+    def __init__(self, axes, crs, draw_labels=False,
```

Comment:
This is changing the `Gridlines` API. Thoughts, @pelson ?

Reply:
Done.